src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -151,11 +151,7 @@ public void visit(NodeTraversal t, Node n, Node parent) {
         }
 
         Node fnNode = NodeUtil.getEnclosingFunction(n);
-        Node fnParent = fnNode.getParent();
-
-        // Use the QualifiedName if the function is on an object/namespace: `foo.moreFoo()`;
-        // otherwise, use the string on the node `function foo()` => `foo`.
-        String fnName = fnParent.isGetProp()? fnParent.getQualifiedName() : fnParent.getString();
+        String fnName = getFnName(fnNode);
```

Comment:
This is the enclosing function, not a function call, so I don't think that's true?